src/components/typography/types.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import React from 'react';
+
+import Types from './types';

```

Comment:
I struggle with this being calls `types` since the name is typically reserved for files that container typescript type definitions, e.g. `type color = 'red' | 'blue'` (although it does have no functional impact).  Perhaps `TypePage` or `TypographyPage` are better names.

Reply:
The name has changed